docker/docker-compose.yml

```diff
@@ -126,8 +126,6 @@ services:
       dockerfile: Dockerfile
     environment:
       - USERS_PORT
-    ports:
```

Comment:
Why change this?

Reply:
`development.yml` and `production.yml` have the ports defined in it (`6666:6666` in dev mode and `6666` in production mode), so having this defined in docker-compose is not necessary anymore. I know that it breaks the convention used throughout the file but having things specified this way allows me to customize the environment when developing or deploying the users microservice.